server/accounts.go

```diff
@@ -4189,33 +4190,41 @@ func placeHolderIndex(token string) ([]int, int32, error) {
 			return []int{tp}, -1, nil
 		}
 
-		// New 'moustache' style mapping
+		// New 'mustache' style mapping
 		// wildcard(wildcard token index) (equivalent to $)
 		args := getMappingFunctionArgs(wildcardMappingFunctionRegEx, token)
 		if args != nil {
+			if len(args) == 1 && args[0] == "" {
```

Comment:
nit: Derek likes to use `_EMPTY_` instead of `""`

Reply:
Done